storage/tianmu/handler/tianmu_handler.cpp

```diff
@@ -548,10 +548,7 @@ int TianmuHandler::delete_row([[maybe_unused]] const uchar *buf) {
  Called from sql_select.cc by JOIN::rein*it.
  Called from sql_union.cc by st_select_lex_unit::exec().
  */
-int TianmuHandler::delete_all_rows() {
-  DBUG_ENTER(__PRETTY_FUNCTION__);
```

Comment:
DBUG_ENTER, DBUG_RETURN, we think that should be kept

Reply:
Yes, it should be kept